ParcelDistributionCenter/ParcelDistributionCenter.Web/Program.cs

```diff
@@ -10,7 +10,7 @@ public static void Main(string[] args)
 
             // Add services to the container.
             builder.Services.AddControllersWithViews();
-            builder.Services.AddScoped<IMemoryRepository, MemoryRepository>();
+            builder.Services.AddScoped<MemoryRepository>();
```

Comment:
Podobnie tutaj, czemu usunąłeś ten interfejs? Bo chyba to było dodane w założeniu korzystania z Dependency Injection

Reply:
Interfejs nie chciał działać po zmianie klasy na statyczną. Potem zmieniłem Interfejs i go dodałem od nowa.